packages/gatsby-theme/src/components/Contact/ContactHeader.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types"
+import RichText from "../RichText"
```

Comment:
Please add RichText component to the [component index file](https://github.com/octahedroid/drupal-gatsby-theme/blob/master/packages/gatsby-theme/src/components/index.js) so you can update this import 
```suggestion
import { RichText } from "ui"
```